api/tests/functional-tests/backend/metrics/test_detection.py

```diff
@@ -2292,11 +2292,11 @@ def test__convert_annotations_to_common_type(db: Session):
     mask[ymin:ymax, xmin:xmax] = True
 
     pts = [
-        (xmin, ymin),
-        (xmin, ymax),
-        (xmax, ymax),
-        (xmax, ymin),
-        (xmin, ymin),
+        (float(xmin), float(ymin)),
```

Comment:
is this change necessary? seems weird that updating the mistral client would require a change in this output